lib/impls/aetherus.ex

```diff
@@ -5,6 +5,14 @@ defmodule PP.Impls.Aetherus do
           {partition1 :: [pos_integer()], partition2 :: [pos_integer()]}
         }
   def solve(nums) do
+
+    # Reversely sort the nums
+    # so that subset-sum can converge faster
+    nums =
+      nums
+      |> Enum.sort()
+      |> Enum.reverse()
```

Comment:
This could probably be better expressed by directly sorting in reverse:

```elixir
|> Enum.sort(:desc)
```

Reply:
Thanks. Good to know.